packages/web/src/services/audius-backend/WithdrawUSDC.ts

```diff
@@ -0,0 +1,224 @@
+import { SolanaWalletAddress } from '@audius/common'
+import { SwapMode } from '@jup-ag/core'
+import {
+  ASSOCIATED_TOKEN_PROGRAM_ID,
+  Token,
+  TOKEN_PROGRAM_ID
+} from '@solana/spl-token'
+import { LAMPORTS_PER_SOL, PublicKey, Transaction } from '@solana/web3.js'
+
+import { JupiterSingleton } from 'services/audius-backend/Jupiter'
+import { waitForLibsInit } from 'services/audius-backend/eagerLoadUtils'
+
+import { getRootSolanaAccount, getSolanaConnection } from './BuyAudio'
+
+// @ts-ignore
+const libs = (): AudiusLibs => window.audiusLibs
```

Comment:
Second thought:

I'm wondering if we should discuss a plan for what our direction is for libs/backend stuff. I only mention it because this file differs from the pattern in `solana.ts` where a backend instance is passed in, each function that needs lib is written in dependency injection style, and functions wait on libs directly. In this file, we're reading it directly off the window instead of dependency-injecting the backend instance.
I can see pros and cons to both ways. And I think ultimately we're trying to get to a place where we can have most code use a well-typed libs instance that doesn't require using overrides (`!`) and ignores in various places to get around the fact that we can't be sure the instance is correctly initialized 😞 .

One additional argument for the DI route, though it's a bit tedious, is that we aren't directly bound to reading something off the window. Which would make things slightly more testable if/when we get to that point.

Anyway, a lot of text to say that maybe if we aren't going to actually use a backend instance here, we should move it out of the `audius-backend` subfolder and up to its own folder in `services`. 

Thoughts?
cc @rickyrombo

Reply:
good call, I think most of these should probably live under a `solana` folder instead